nengo/utils/numpy.py

```diff
@@ -7,6 +7,12 @@
 import numpy as np
 
 from ..exceptions import ValidationError
+try:
+    from scipy.linalg import expm  # noqa: F401 pylint: disable=unused-import
```

Comment:
I think my intention was in case Scipy improves their expm function in any way, we use the best. But I think that's unlikely, and probably worth the simplicity of always using the vendorized one.

Reply:
Gotcha, will change to using the vendorized one :ok_hand: